test/helpers/types/types.go

```diff
@@ -343,7 +337,25 @@ func StrictlyParseLogs(in string, logs interface{}) error {
 
 type Logs []AllLog
 
-func (t *AllLog) ElapsedEpoc() float64 {
+func ToJsonLogs(logs []string) string {
```

Comment:
Replace with existing `test.JSONLine()`, does the same thing.

Reply:
this didnt give the same result. so didnt change

Reply:
Of course - line breaks. Sorry.